### PR TITLE
Prepare 1.8.2 release

### PR DESCRIPTION
**Pull request by iwysiu**

### CODE REVIEW

**Comment by fridgepoet** [comment]
Same as https://github.com/grafana/timestream-datasource/pull/233#issuecomment-1534264974

**Comment by fridgepoet** [comment]
Athena data source had no issue with the freshly-released grafana-plugin-sdk-go v0.161.0 -- sorry it's a pain, but would you use that instead?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.9.0
+
+- Update grafana-aws-sdk version
+- Security: Upgrade Go in build process to 1.20.4
+- Revert grafana-plugin-sdk-go version to 0.149.0 to fix connection issues. Should be same behavior as behavior with no known issues.
```
Comment (fridgepoet) [inline CHANGELOG.md:7]: Oops I probably left that typo in my original changelog edit -- but if we upgrade, we won't need that part!

Comment (iwysiu) [reply CHANGELOG.md]: removed!

**Review by kevinwcyu — APPROVED** [review]

**Review by fridgepoet — APPROVED** [review]

**Pull request merged**